src/components/Form/FormSection.jsx

```diff
@@ -14,28 +18,51 @@ import Typography from '@mui/material/Typography';
  * that consist of title and children (see {@link formSectionProps})
  * @param {string} Props.headingId - The `id` attribute of the element containing the `title`
  * @param {string} Props.title - Title of form section
- * @param {React.ReactElement} Props.children - JSX Element of the wrapped form
- * @returns {React.JSX.Element} - The FormSection Component
+ * @param {React.ReactElement} Props.children - The child elements to be rendered inside the main content area
+ * @returns {React.JSX.Element} The FormSection Component
  */
-const FormSection = ({ title, headingId, children }) => (
-  <Box
-    sx={{
-      marginTop: 1,
-      display: 'flex',
-      flexDirection: 'column',
-      justifyContent: 'center',
-      alignItems: 'center',
-      mx: '3px',
-      padding: '16px',
-      minWidth: '50%',
-      maxWidth: '75dvw'
-    }}
-  >
-    <Typography variant="h2" align="center" mb={2} sx={{ fontSize: '20px' }} id={headingId}>
-      {title}
-    </Typography>
-    {children}
-  </Box>
-);
+const FormSection = ({ title, headingId, children }) => {
+  const theme = useTheme();
+  // eslint-disable-next-line no-unused-vars
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+
+  const truncatedText = title ? truncateText(title) : '';
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        padding: '20px',
+        // minWidth: '50vw',
+        // minWidth: isSmallScreen ? null : '50dvw',
+        // maxWidth: isSmallScreen ? '100dvw' : '75dvw',
```

Comment:
Given we're making the components swap to a drawer depending on screen width, I think it's fair to say we can remove these older styles.